src/app/auth/onboarding/styles.tsx

```diff
@@ -4,25 +4,34 @@ import colors from '../../../styles/colors';
 
 export default StyleSheet.create({
   container: {
-    paddingVertical: 63,
+    paddingVertical: 66,
     paddingLeft: 43,
     paddingRight: 44,
+    backgroundColor: 'white',
   },
   subtext: {
     color: colors.darkGrey,
     marginLeft: 8,
   },
-  h1: {
-    marginTop: 66,
-  },
   body1: {
     marginTop: 26,
   },
   info: {
     flex: 1,
     flexDirection: 'row',
     marginTop: 12,
-    marginBottom: 16,
     width: 250,
   },
+  bottomContainer: {
+    marginTop: 112,
+  },
+  updateProfileButton: {
+    marginTop: 30,
```

Comment:
You can remove the margins here because the flex space evenly should take care of that